### DISCUSSION TITLE
Java doc equiv of this C# patching guide please!

### DISCUSSION THREAD

**Discussion by genachka** [ravendb/ravendb#14761] [General]
This is the one I found for C# and please make one for Java: https://ravendb.net/docs/article-page/5.4/csharp/client-api/operations/patching/single-document

And in general, please improve the Java docs to be equal to C#. So many examples and details are missing in the Java docs.

**Comment by garayx** [comment]
Hi

Thanks for noticing this, we will update our docs,

https://issues.hibernatingrhinos.com/issue/RDoc-2248/Create-java-version-of-client-apioperationspatchingsingle-document
